docs/api/api.yml

```diff
@@ -1059,12 +1059,15 @@ components:
       properties:
         scheme:
           type: string
+          description: "Possible values: degrees, RD."
```

Comment:
seems incorrect. The following are the correct values:

* `RD` -> `http://www.opengis.net/def/crs/EPSG/0/28992`
* `degrees` -> `http://www.opengis.net/def/crs/EPSG/0/4326`

@jo-pol please verify that this is reflected in the code!

Reply:
Actually, the api and DDM.xsd allow everything. The api maps the values `RD` and `degrees` (case sensitive) to the URI's for the DDM.

https://github.com/DANS-KNAW/easy-deposit-api/blob/5b20f651586305350e754bf8fe868943798b200a/src/main/scala/nl.knaw.dans.easy.deposit/docs/dm/Spatial.scala#L29-L35
https://github.com/DANS-KNAW/easy-deposit-api/blob/5b20f651586305350e754bf8fe868943798b200a/src/test/scala/nl.knaw.dans.easy.deposit/docs/DDMSpec.scala#L310-L317

Reply:
Now that I looked into the code: since the api doesn't validate, should we really document it here? Did not try whether ingest flow adds validation to the `ddm.xsd`

Reply:
@lindareijnhoudt what do you think? Should we document these values here?

Please note, as far as I can tell, the value for `scheme` is **never** validated in `easy-deposit-api`, `easy-ingest-flow` or any of the other services related to ingest. Hence, you can basically put whatever in here...

Reply:
yes, the idea there being that you are allowed to use whatever coordinate-system you want. only if it is provided in one of these two, we can understand what is meant, and propagate accordingly.

so yes, it should be documented here, to tell that these are understood by EASY

Reply:
@rvanheest Note that possible has been changed into recommended.